db/migrate/20231010180442_create_mentee_questionnaires.rb

```diff
@@ -0,0 +1,25 @@
+class CreateMenteeQuestionnaires < ActiveRecord::Migration[7.0]
+  def change
+    create_table "mentee_questionnaires", id: :binary, force: :cascade do |t|
+      t.binary "respondent_id", limit: 16, null: false, foreign_key: true
+      t.string :name, null: false
+      t.string :country, null: false
+      t.string :city, null: false
```

Comment:
Don't need these here they're on user

Reply:
I may have blithely added them to mentor questionnaire! Oops.

Reply:
ah I overlooked here. Left out email but not country and city 😅.

Will get a separate PR with a migration to remove the rows from the table.

Reply:
You can make the changes in here:

On your local branch:

1. `rails db:rollback`
2. Make the changes to this migration
3. `rails db:migrate`

Reply:
Sorry wasn't specific, I meant a separate migration to also remove it from the mentor_questionnaire model. I've removed it from this one in this PR.